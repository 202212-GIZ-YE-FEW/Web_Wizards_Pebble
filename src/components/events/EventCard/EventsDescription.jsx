const EventsDescription = ({ description }) => {
    const shortDescription = description?.slice(0, 100) + "...";

    return <p>{shortDescription}</p>;
};

export default EventsDescription;
