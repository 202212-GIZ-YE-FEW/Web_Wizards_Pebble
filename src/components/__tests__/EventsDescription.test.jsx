import renderer from "react-test-renderer";

import EventsDescription from "../events/EventCard/EventsDescription";

it("renders correctly", () => {
    const tree = renderer.create(<EventsDescription />).toJSON();
    expect(tree).toMatchSnapshot();
});
