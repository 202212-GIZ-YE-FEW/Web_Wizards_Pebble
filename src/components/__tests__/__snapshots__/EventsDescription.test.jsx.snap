// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<p
  className="text-sm"
>
  <span
    className="hidden sm:inline"
  >
    undefined...
  </span>
  <span
    className="inline sm:hidden"
  >
    undefined...
  </span>
</p>
`;
